packages/graphql-codegen-golang/src/generator.ts

```diff
@@ -267,7 +274,7 @@ export class GolangGenerator {
         nonNull
       )
     }
-    throw new Error(`field type "${type}" not supported!`)
+    throw new Error(`field type "${type.name.value}" not supported!`)
```

Comment:
This is unrelated change, but I've found it easier to spot what field isn't supported.